ako-operator/helm/ako-operator/templates/ako-operator-manager-role.yaml

```diff
@@ -53,4 +53,10 @@ rules:
   verbs: ["get", "watch", "list"]
 - apiGroups: ["apiextensions.k8s.io"]
   resources: ["customresourcedefinitions","customresourcedefinitions/status"]
-  verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
\ No newline at end of file
+  verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
```

Comment:
@DixitAakash: We can skip this as we are not claiming support for these.

Reply:
Removed.

Reply:
@DixitAakash: we cam remove multiclusteringresses, serviceimports fields? @swathinsankaran: Please check this if we remove these fields will it impact or not.

Reply:
I think I missed removing these. These changes were added by me and I don't think they will be needed. Will remove it and push the changes.

Reply:
Removed.